grpcclient/lc_client.go

```diff
@@ -166,14 +167,20 @@ func NewLcClient(setters ...LcClientOption) *LcClient {
 		setter(cli)
 	}
 
+	retryOpts := []grpc_retry.CallOption{
+		grpc_retry.WithMax(3),
+		grpc_retry.WithBackoff(grpc_retry.BackoffExponential(500 * time.Millisecond)),
```

Comment:
Did you consider `BackoffExponentialWithJitter` ? Such jitter could protect server side if there are multiple client that start retry at the same time.

Reply:
added